home/views.py

```diff
@@ -1,5 +1,21 @@
 from django.shortcuts import render
 from django.http import JsonResponse
+from rest_framework import viewsets
+from .serializers import InfoSerializers
+from .models import Information
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework import status
+import json
+from django.http import QueryDict,Http404
+
 
 def home(request):
-  return JsonResponse(dict(message='Welcome to the Phonebook API'), status=200)
\ No newline at end of file
+  return JsonResponse(dict(message='Welcome to the Phonebook API', ticket="12345"), status=200)
+
+class Info(APIView):
+	def get(self, request, *args, **kwargs):
```

Comment:
what does this method do? it would be nice to use comments to give a detailed information of its arguments, what it does and what it returns.

example:
def days_until_launch(current_day, launch_day):
    """"Returns the days left before launch.
    
    current_day (int) - current day in integer
    launch_day (int) - launch day in integer
    """
    days_left=launch_day - current_day
    
    if days_left > 0:
        return days_left
    return 0